src/utils/apiHandler/likedVideoHandler.js

```
@@ -0,0 +1,46 @@
+import axios from "axios"
+import { GET_LIKED_VIDEOS } from "../../reducer/video/videoListConstant"
+
+const encodedtoken = localStorage.getItem("token");
+const config = {
+  headers: {
+    authorization: encodedtoken,
+  },
+};
+
+export const getLikeVideos = async(dispatch) => {
+    try {
+        const response = await axios.get("/api/user/likes", config)
+        dispatch({type: GET_LIKED_VIDEOS, payload: response.data.likes})
+        console.log(encodedtoken)
+    }
+    catch(err) {
+        console.log(err)
```

You can throw the error console.errror(error)